### ISSUE TITLE
Oligodendroglioma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6765]
**Preferred term label:**
Oligodendroglioma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of WHO grade II or III glioma.

**Parent term (use hpo.jax.org/app)**
Glioma HP:0009733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM # 137800
GLIOMA SUSCEPTIBILITY 1

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
adding. Updated the definition to provide more context

Oligodendroglioma is a type of diffusely infiltrating glioma and constitutes approximately 5% of primary intracranial tumors. They often involve the cortical gray matter and are most commonly seen in the frontal lobes. OGs are generally low grade WHO grade II neoplasms that are slow-growing tumors and have a favorable treatment response when compared to other gliomas. Grade III anaplastic OG is a more malignant form of the tumor which portends a less favorable prognosis and may occur de novo or as degeneration from the lower grade OG.
